resources/inspectionDescriptions/PreferenceXmlInspections.html

```diff
@@ -0,0 +1,18 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<html>
+<body>
+<p>
+  Validates classes and interfaces for  attributes "for" and the "type" for the "preference" tag
```

Comment:
@Iamwade, please remove extra space after the first "for". And if you can, please, formulate this sentence differently. For now, it is hard to read.

Reply:
Fixed.

Reply:
Thank you!